.github/workflows/remove_dev_deployment.yml

```
@@ -0,0 +1,23 @@
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  remove-dev-deployment:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Derive TAG_NAME from branch name
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)